src/swapnaWJun20/Demo.java

```diff
@@ -0,0 +1,12 @@
+package swapnaWJun20;
+
+public class Demo {
+	
+	void Display(){
```

Comment:
Method name should be start with small letter.

Reply:
Done